.github/workflows/on_release_publish.yml

```diff
@@ -10,6 +10,24 @@ on:
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
+  # Run visual tests with Chromatic
```

Comment:
Is this repeated in `ui-tests.yml` ?

Reply:
Yes, it is.

Reply:
I want to run the visual tests when opening a PR and also when pushing to main. Maybe I can update this: 
```
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]

```

?

Reply:
Updated to this:

```
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
  push:
    branches:
      - main
```